content/2021-11-05-introducing-drogue-server/index.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Introducing Drogue Cloud Server"
+extra.author = "lulf"
++++
+
+We love Kubernetes and the standardized API it brings to deploying applications in the cloud. For running a local instance of Drogue Cloud, [minikube](https://blog.drogue.io/minikube-roundtrip/) is already a great alternative. But sometimes, you just want to have an easy getting started experience, to reduce turnaround time when testing changes to Drogue Cloud itself, or for running in environments where Kubernetes is not available.
+
+Read on to learn how you can run Drogue Cloud standalone.
+
+<!-- more -->
+
+# Appreciating Kubernetes
+
+Drogue Cloud on Kubernetes automatically installs some dependencies of Drogue Cloud such as:
+
+* [PostgreSQL](https://www.postgresql.org/)
+* [Keycloak](https://www.keycloak.org/)
+* [Kafka](https://kafka.apache.org/)
+
+However, running outside of Kubernetes also means that you need to run or configure these services yourself. You can do this by installing the software manually and ensuring they are accessible locally. You can also rely on (tada!) containers using `docker-compose` or `podman-compose` if it is available on your platform, and we provide a [compose file](https://github.com/drogue-iot/drogue-cloud/blob/main/server/container-compose.yml) just for that purpose.
+
+
+# Installing The Binary
+
+One of the goals with running Drogue Cloud locally was to encapsulate the functionality in a single binary, `drogue-cloud-server`. You can [build it yourself](https://github.com/drogue-iot/drogue-cloud/tree/main/server#building), or you can [download prebuilt binaries]() for your platform.
+
+# Running
+
+Once installed, it's time to run. You can find the detailed instructions in [the book](https://book.drogue.io/drogue-cloud/dev/deployment/single-binary.html). It's important to note that for the time being, the Kafka, Keycloak and PostgreSQL instance must be reachable on localhost where the server is running.
+
+Once you have the dependencies running, we can start the server:
+
+```rust
+./drogue-server run --enable-all
+```
+
+When starting, the server will perform the following steps (in order):
+
+* Connect to PostgreSQL and run schema migration
+* Connect to Keycloak and create an OIDC client
+* Launch the services specified for the run command
+
+Using `--enable-all` will run all Drogue Cloud services supported. At the time of writing, this is:
+
+* REST API
+* Device registry
+* Device authentication
+* User authentication
+* HTTP endpoint
+* MQTT endpoint
+
+We plan to support all the Drogue Cloud services.
```

Comment:
Maybe "expose" instead of "support" or such, since "support" has a very distinct definition to The Company?